### ISSUE TITLE
Endless migration generation in Django 1.7

### ISSUE THREAD

**Issue by amadornimbis** [jrief/djangocms-cascade#70]
I have encountered a problem in which I can't run `./manage.py migrate` in my application because Django constantly detects there are unmigrated changes in cmsplugin_cascade. The only differences in each of these migrations appears to be the ordering of the list of `choices` related to `model_name='pluginextrafields'`. For example, here are migrations 0002, 0003, and 0004 that are generated in my project for cmsplugin_cascade.

0002_auto_20150429_1310:
```
class Migration(migrations.Migration):

    dependencies = [
        ('cmsplugin_cascade', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='pluginextrafields',
            name='plugin_type',
            field=models.CharField(db_index=True, max_length=50, verbose_name='Plugin Name', choices=[(b'BootstrapButtonPlugin', 'Bootstrap Button'), (b'SimpleWrapperPlugin', 'Bootstrap Simple Wrapper'), (b'HorizontalRulePlugin', 'Bootstrap Horizontal Rule'), (b'BootstrapRowPlugin', 'Bootstrap Row')]),
            preserve_default=True,
        ),
    ]
```

0003_auto_20150429_1346:
```
class Migration(migrations.Migration):

    dependencies = [
        ('cmsplugin_cascade', '0002_auto_20150429_1310'),
    ]

    operations = [
        migrations.AlterField(
            model_name='pluginextrafields',
            name='plugin_type',
            field=models.CharField(db_index=True, max_length=50, verbose_name='Plugin Name', choices=[(b'HorizontalRulePlugin', 'Bootstrap Horizontal Rule'), (b'BootstrapButtonPlugin', 'Bootstrap Button'), (b'BootstrapRowPlugin', 'Bootstrap Row'), (b'SimpleWrapperPlugin', 'Bootstrap Simple Wrapper')]),
            preserve_default=True,
        ),
    ]
```

0004_auto_20150429_1346.py:
```
class Migration(migrations.Migration):

    dependencies = [
        ('cmsplugin_cascade', '0003_auto_20150429_1346'),
    ]

    operations = [
        migrations.AlterField(
            model_name='pluginextrafields',
            name='plugin_type',
            field=models.CharField(db_index=True, max_length=50, verbose_name='Plugin Name', choices=[(b'HorizontalRulePlugin', 'Bootstrap Horizontal Rule'), (b'SimpleWrapperPlugin', 'Bootstrap Simple Wrapper'), (b'BootstrapRowPlugin', 'Bootstrap Row'), (b'BootstrapButtonPlugin', 'Bootstrap Button')]),
            preserve_default=True,
        ),
    ]
```

These are only the first three migrations I generated. As far as I can tell, it will keep doing this endlessly. This problem is occurring after an upgrade to Django 1.7.7 and djangocms-cascade 0.4.4. I never observed this problem when working with Django 1.6.x and djangocms-cascade 0.4.3. Is this a known bug or perhaps a problem with my cascade configuration? Thanks.

**Comment by jrief** [comment]
For the moment I'm still stuck on Django-1.6.

Since cascade is not shipped with Django-1.7 migrations yet, I assume you created the above migrations using ``./manage.my makemigrations cmsplugin_cascade``. This means your problem happens then.
Hmmm, isn't that a Django problem? Cascade just adds a few not very spectacular models.

**Comment by rfleschenberg** [comment]
I think this issue is outdated (Django 1.7 is not supporetd by Django anymore). Closing.
